test/support/test_reporter.ex

```diff
@@ -55,9 +55,8 @@ defmodule Elixometer.TestReporter do
   def options_for(metric_name) do
     Application.get_env(:elixometer, :reporter)
     |> :exometer_report.list_subscriptions
-    |> Enum.filter_map(
-      fn {name, _, _, _}  -> name == metric_name end,
-      fn {_, _, _, extra} -> extra end)
+    |> Enum.filter(fn {name, _, _, _}  -> name == metric_name end)
+    |> Enum.map(fn {_, _, _, extra} -> extra end)
```

Comment:
I think this would be more idiomatic if it used `Enum.find_value` but it won't work if `extra` can be `nil` or `false` (I think it cant be here though).